QUICKSTART.md

```diff
@@ -47,10 +47,16 @@ Tracer tracer =
 To create a basic span, you only need to specify the name of the span.
 The start and end time of the span is automatically set by the OpenTelemetry SDK.
 ```java
-Span span = tracer.spanBuilder("SpanName").startSpan();
-// your use case
-...
-span.end();
+Span span = tracer.spanBuilder("my span").startSpan();
+try (Scope scope = tracer.withSpan(span)) {
+	// your use case
+	...
+} catch (Throwable t) {
+    Status status = Status.UNKNOWN.withDescription("Change it to your error message");
```

Comment:
```suggestion
    Status status = Status.UNKNOWN.withDescription(t.getMessage());
```

Reply:
@arminru i'm not sure about it, t.getMessage() can provide a little explanation and user may want to change this to custom string including or excluding t.getMessage or even not exposing inner error

Reply:
Alright, I'm fine either way.